spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-kubernetes.adoc

```diff
@@ -233,47 +251,49 @@ In the event you would not like to deploy metrics collection using Prometheus an
           grafana-info:
             url: 'http://grafana:3000'
 ----
+====
 
+[[getting-started-kubernetes-deploy-services-and-data-flow-create-df-role-bindings-service-account]]
+====== Create Data Flow Role Bindings and Service Account
 
-3) Create Data Flow Role Bindings and Service account
-
-IMPORTANT: Since version 1.9, the latest releases of Kubernetes have enabled https://kubernetes.io/docs/admin/authorization/rbac/[RBAC] on the api-server.
-If your target platform has RBAC enabled, you must ask a `cluster-admin` to create the `roles` and `role-bindings` for you before deploying the Data Flow server.
-They associate the Data Flow service account with the roles it needs to be run with.
-
-To create Role Bindings and Service account:
+IMPORTANT: Since version 1.9, the latest releases of Kubernetes have enabled https://kubernetes.io/docs/admin/authorization/rbac/[RBAC] on the API server.
```

Comment:
This note may not be necessary anymore. 1.9 version is not a supported distro anymore anyway; K8s is at 1.13 right now.

Reply:
@chrisjs: WDYT? ^^

Reply:
well, depends on what we consider the platform for "supported distro". IE: OSS k8s, GKE, etc.

since we have primarily used GKE as a base and stayed within 3 versions, it looks like they now support 1.12 clusters so id say based on that, yes 1.9 we would drop

Reply:
No point stating it in the docs. +1 to dropping it.